core/src/main/java/com/klaytn/caver/account/WeightedPublicKey.java

```diff
@@ -0,0 +1,87 @@
+package com.klaytn.caver.account;
+
+import com.klaytn.caver.utils.AccountKeyPublicUtils;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+public class WeightedPublicKey {
+
+    /**
+     * ECC Public Key value with "SECP-256k1" curve.
+     * This String has following format.
+     * 1. Uncompressed format : 0x{Public Key X point}||{Public Y point}
+     * 2. Compressed format : 0x{02 or 03 || Public Key X}
+     */
+    private String publicKey;
+
+    /**
+     * The weight of key
+     */
+    BigInteger weight;
+
+    public static final int OFFSET_WEIGHT = 0;
+    public static final int OFFSET_PUBLIC_KEY = 1;
```

Comment:
Do we need these?

Reply:
This const values used encodedToBytes() return data.
I think when accessing array element, Using string is more human readable than index value.

Reply:
Ah! okayokay. I understood now! Thank you for explanation :)